### PR TITLE
Updating information about Edge

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary
Updating information regarding Edge in the [Building a cross-browser extension](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Build_a_cross_browser_extension) article.

Mote: advice being sought with regard to including information on manifest V3.

#### Motivation
Information Edge out of date and related to the pre-Chromium version.

#### Related issues
Fixes Out of date Edge information in Build a cross-browser extension [#1462](https://github.com/mdn/content/issues/1462)

#### Metadata

This PR…
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.md [hunk 1]
```diff
@@ -76,32 +78,32 @@ So, for example, this `manifest.json` code makes the polyfill available to your
...
-There are other polyfill options but, at the time of writing, none provide the coverage of the WebExtension browser API Polyfill. So, where you haven’t targeted Firefox as your first choice, your options are to accept the limitations of alternative polyfills, port to Firefox and add cross-browser support, or develop your own polyfill.
+There are other polyfill options. However, at the time of writing, none of the other options provide the coverage of the WebExtension browser API Polyfill. So, where you haven’t targeted Firefox as your first choice, your options are to accept the limitations of alternative polyfills, port to Firefox and add cross-browser support, or develop your own polyfill.
 
 ### API function coverage
 
 The differences in the API functions offered in each of the four main browsers fall into three broad categories:
 
 1.  **Lack of support for an entire function.**
-    For example, at the time of writing, Edge didn’t provide support for the [`privacy`](/en-US/docs/Mozilla/Add-ons/WebExtensions/API/privacy#browser_compatibility) function.
+    For example, at the time of writing, Edge didn’t support the {{WebExtAPIRef("browserSettings")}}.
```
Comment (sideshowbarker) [inline files/en-us/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.md:92]:
```suggestion
    For example, at the time of writing, Edge didn’t support the {{WebExtAPIRef("browserSettings")}} function.
```

File: files/en-us/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.md [hunk 2]
```diff
@@ -76,32 +78,32 @@ So, for example, this `manifest.json` code makes the polyfill available to your
...
 ### API function coverage
 
 The differences in the API functions offered in each of the four main browsers fall into three broad categories:
 
 1.  **Lack of support for an entire function.**
-    For example, at the time of writing, Edge didn’t provide support for the [`privacy`](/en-US/docs/Mozilla/Add-ons/WebExtensions/API/privacy#browser_compatibility) function.
+    For example, at the time of writing, Edge didn’t support the {{WebExtAPIRef("browserSettings")}} function.
 2.  **Variations in the support for features within a function.**
-    For example, at the time of writing, Firefox doesn’t support the notification function method [`onButtonClicked`](/en-US/docs/Mozilla/Add-ons/WebExtensions/API/notifications/onButtonClicked#browser_compatibility) while Firefox is the only browser that supports [`onShown`](/en-US/docs/Mozilla/Add-ons/WebExtensions/API/notifications/onShown#browser_compatibility).
+    For example, at the time of writing, Firefox doesn’t support the notification function method {{WebExtAPIRef("notifications.onButtonClicked")}} while Firefox is the only browser that supports {{WebExtAPIRef("notifications.onShown")}}.
```
Comment (sideshowbarker) [inline files/en-us/mozilla/add-ons/webextensions/build_a_cross_browser_extension/index.md:94]:
```suggestion
    For example, at the time of writing, Firefox doesn’t support the notification function method {{WebExtAPIRef("notifications.onButtonClicked")}}, while Firefox is the only browser that supports {{WebExtAPIRef("notifications.onShown")}}.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by rebloor** [comment]
Thanks @sideshowbarker

**Review by Rob--W — COMMENTED** [review]
Just a few comments, and not a full review since the PR has already been closed (it would have been nice if the PR didn't get merged until I have had a chance to post a review).

While the focus of this PR seems to be Edge, it's also worth noting that the documentation about Safari are also out-of-date.

**Pull request merged**